tools/gromacs/trj.xml

```diff
@@ -0,0 +1,174 @@
+<tool id="gmx_trj" name="Modify/convert GROMACS trajectories" version="@VERSION@">
+    <description>using trjconv and trjcat</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements" />
+
+    <command detect_errors="exit_code"><![CDATA[
+
+    #if $trj.trj_op == 'trjcat':
+        mkdir trajs &&
+        #for $value, $file in enumerate($trj_input):
+            ln -s '$file' trajs/traj_${value}.${file.ext} &&
+        #end for
+
+        gmx trjcat
+            -f trajs/traj* ## here we assume that they all have the same ext, probably reasonable
+            $trj.cat
+            -e '$time.e'
+
+    #elif $trj.trj_op == 'trjconv':
+        ln -s '$ndx_input' ./index.ndx &&
+        ln -s '$str_input' ./str.${str_input.ext} &&
+        ln -s '$trj_input' ./traj.${trj_input.ext} &&
+        #if $trj.center:
+            echo '$trj.index_center $trj.index_output' | gmx trjconv
+        #else
+            echo '$trj.index_output' | gmx trjconv
+        #end if
+            -f ./traj.${trj_input.ext}
+            -s ./str.${str_input.ext}
+            -n ./index.ndx
+            $trj.center
+            -pbc $trj.pbc
+            -ur $trj.ur
+            -boxcenter $trj.boxcenter
+            -e #echo str($time.e).replace('-1', '1000000000000000')#
```

Comment:
can we use a if here to make it more explicit?
Also you might want to mention the implicit default in the help of the parameter?